tests/backend_test.py

```diff
@@ -151,10 +155,8 @@ def test_measures() -> None:
 
 @pytest.mark.skipif(skip_remote_tests, reason=REASON)
 def test_noise() -> None:
```

Comment:
We should define the provider in the same way the backends are created for the remote tests.